lib/tasks/admin.rake

```diff
@@ -32,4 +32,28 @@ namespace :admin do
 
     exit 0
   end
+
+  desc 'Create an super admin account'
+  task :super_admin, %i[name email password] => :environment do |_task, args|
+    super_admin_email = "superadmin-#{args[:email]}"
+
+    user = User.create(
+      name: args[:name],
+      email: super_admin_email,
+      password: args[:password],
+      role: Role.find_by(name: 'SuperAdmin'),
```

Comment:
Umm, Why don't we be more explicit and `find_by(:name, :provider)`?

Reply:
Yeah we can do that